src/server/verification/api/__tests__/__util__/index.js

```diff
@@ -65,6 +66,21 @@ export default zoomServiceMock => {
       }
     })
 
+  const mockLivenessError = () =>
+    zoomServiceMock.onPost('/search').reply(400, {
+      meta: {
+        code: 400,
+        ok: false,
+        m: { zb: 2256, st: 1594565626.252231, bt: null, bp: null, zw: 2545, zp: 2528 },
+        subCode: 'unableToProcess',
+        mode: 'dev',
+        message: '3D FaceMaps that are used with Search APIs must have had Liveness Proven.'
+      },
+      data: {
+        isLive: false
```

Comment:
1. isLive doesn't returns from the ZoOm server. **our processor** passes this flag to the session subscriber and adds to the enrollmentResult
2. in case of error ZoOm doesn't returns 'data' property - just 'meta' - see other error cases